src/components/Home.js

```diff
@@ -0,0 +1,136 @@
+import React, { useState } from 'react';
+import { NavLink } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import { showPlanet } from '../redux/home/homeSlice';
+import logo from '../assets/solar-system.png';
+import earth from '../assets/worldwide.png';
+import venus from '../assets/venus.png';
+import mars from '../assets/mars.png';
+import mercury from '../assets/mercury.png';
+import saturn from '../assets/saturn.png';
+import neptune from '../assets/neptune.png';
+import jupiter from '../assets/jupiter.png';
+import uranus from '../assets/uranus.png';
+
+const Home = () => {
```

Comment:
- [Optional] It is always advisable to split the big complex component into a few smaller simpler components so it is easier to understand :+1: